pages/dkp/kommander/2.1/security/policy-controls/gatekeeper/index.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: layout.pug
+navigationTitle: Enforce policies using Gatekeeper
+title: Enforce policies using Gatekeeper
+menuWeight: 24
+excerpt: Learn how to enforce policies using Gatekeeper
+beta: false
+enterprise: false
+---
+
+<!-- markdownlint-disable MD024 -->
+
+[Gatekeeper][gatekeeper] is the policy controller for Kubernetes, allowing organizations to enforce configurable policies using the [Open Policy Agent][opa], a policy engine for Cloud Native environments hosted by CNCF as a graduated-level project.
+
+This tutorial describes how to use Gatekeeper to enforce policies by rejecting non-compliant resources. Specifically, this tutorial describes two constraints as a way to show how to use Gatekeeper as an alternative to [Pod Security Policies][psp]:
+
+- [Prevent the running of privileged pods](#prevent-privileged-pods)
+- [Prevent mounting host path volumes](#prevent-host-path-volumes)
+
+## Before you begin
+Before starting this tutorial, verify the following:
+
+- You must have access to a Linux, macOS, or Windows computer with a supported operating system version.
+
+- You must have a properly deployed and running cluster. For information about deploying Kubernetes with default settings on different types of infrastructures, see the [Choose Infrastructure][choose-infrastructure] topic.
+
+- If you install Kommander with a custom configuration, make sure you enabled gatekeeper.
```

Comment:
@msdolbey @cmastr another pass please, the enable gatekeeper link I used is not accurate since it contains additional configs about http proxy..

Reply:
Oh sure - no probs.

Reply:
So, this reads fine enough for me, and I don't think there are any issues the way it's written, but more of a question to you @cmastr @cwyl02 - 
I think this is fine leaving it a bit up to the user to confirm, but have a quick question if we want to say something like:

You can confirm if the Gatekeeper app is running by running the command:

`kubectl get pods -A | grep gatekeeper`

Or something like that.

Reply:
But, other than that, and I could go either way on that, this is good to me.